docs/using-rucio-client.md

```diff
@@ -173,31 +139,24 @@ invalid/expired/does not have refresh token in the DB, no refresh will
 be attempted.
 
 Example of rucio.cfg file configuration with automatic token refresh:
-
->    **[client]**
->
->    **rucio_host = https://<rucio_host>:443**
->
->    **auth_host = https://<rucio_auth_host>:443**
->
->    **auth_type = oidc**
->
->    **account = <rucio_account_name>**
->
->    **oidc_audience = rucio**
->
->    **oidc_scope = openid profile offline_access**
->
->    **oidc_issuer = wlcg**
->
->    **auth_oidc_refresh_active true**
->
->    **auth_oidc_refresh_before_exp 20**
-
+```bash
```

Comment:
This is not expected to be bash. The rucio.cfg is a file & these can be just code-blocks.